kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/LogSegment.java

```diff
@@ -62,6 +66,29 @@ public LogSegment(
       List<FileStatus> checkpoints,
       Optional<Long> checkpointVersionOpt,
```

Comment:
why do we take this as an argument? can't we find this from the checkpoint files provided?

I know it's already here, just got confused on what it even was, and now wondering why we even take it as an arg

Reply:
In hindsight, yeah, why don't I just remove it!